### PR TITLE
Create 2022-08-24.md

### PR DESCRIPTION
**Pull request by akhsi1**

### CODE REVIEW

File: DailyQuestionHistory/2022-08-24.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+        '4': ["g","h","i"],
+        '5': ["j","k","l"],
+        '6': ["m","n","o"],
+        '7': ["p","q","r","s"],
+        '8': ["t","u","v"],
+        '9': ["w","x","y","z"]
+    };
+    var len = digits.length;
+    for(var letters of map[digits.charAt(0)]){
+        for(var letter of letters){
```
Comment (stanfordpeng) [inline DailyQuestionHistory/2022-08-24.md]: letters is already a single alphabet. Why do we need to double loop it?

File: DailyQuestionHistory/2022-08-24.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    var res = [];
+    for (var i = 0; i < len ; i++){
+        find(i, 0, []);
+    }
+    function find(index, sum, list){
+        var cand = candidates[index];
+        list.push(cand);
+        sum += cand;
+        if ( sum == target ){
+            res.push(list.slice());
```
Comment (stanfordpeng) [inline DailyQuestionHistory/2022-08-24.md]: no need to use the slice here to make a copy as it is not reused anymore

File: DailyQuestionHistory/2022-08-24.md (hunk 1 above)
Comment (stanfordpeng) [reply DailyQuestionHistory/2022-08-24.md]: `for(var letter of map[digits.charAt(0)]){
            build(letter, 0);
        }
`
is enough

**Review by stanfordpeng — CHANGES_REQUESTED** [review]

File: DailyQuestionHistory/2022-08-24.md (hunk 1 above)
Comment (ChengGuang-Li) [inline DailyQuestionHistory/2022-08-24.md]: Eg: 
digits : "23" 
map[digits.charAt(0)]  --> ["a","b","c"]
for(var letters of ["a","b","c"])

❓why is another loop needed here?  for(var letter of letters)

Comment (akhsi1) [reply DailyQuestionHistory/2022-08-24.md]: fixed

File: DailyQuestionHistory/2022-08-24.md (hunk 2 above)
Comment (akhsi1) [reply DailyQuestionHistory/2022-08-24.md]: fixed

**Review by ChengGuang-Li — APPROVED** [review]

**Review by ChengGuang-Li — APPROVED** [review]

**Review by stanfordpeng — APPROVED** [review]

**Pull request merged**